test/DynamicExpresso.UnitTest/LambdaExpressionTest.cs

```diff
@@ -0,0 +1,274 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DynamicExpresso.UnitTest
+{
+	[TestFixture]
+	public class LambdaExpressionTest
+	{
+		private const InterpreterOptions _options = InterpreterOptions.Default | InterpreterOptions.LambdaExpressions;
+
```

Comment:
It could be nice to have some more tests to check what happen with invalid lambda expressions? (like a lambda with wrong parenthesis, or with a typo, or with multiple arrows, ...

And also to verify what happen with lambda expression but without the `LambdaExpressions` flag.

I would just want to ensure that a "readable" `ParseException` is throw.

What do you think?

Reply:
Good idea, I'll add some more tests.